CHANGELOG.rst

```diff
@@ -12,6 +12,7 @@ master (unreleased)
 - Reactivate accidentally skipped ``test_validations.py`` tests
 - Add JSON migrations
 - ``FormidableItem.value`` field size now has no limit (``TextField``)
+- Migrate to PeopleDoc GitHub organization
```

Comment:
I'd add a reference to the #283 issue here

Reply:
Amended.